Katello/katello
# Fixes #33650 - Add filename attribute to generic content units

### What are the changes introduced in this pull request?
This adds a `filename` column to the `katello_generic_content_units` table and makes filename viewable and searchable in the python packages browser UI. The purpose of this is to help differentiate between python packages, as described here: https://projects.theforeman.org/issues/33650
### What are the testing steps for this pull request?
1. Create and sync a python repository with `https://pypi.org` as `upstream url` and `numpy==1.21.1` as `includes`.
2. Go  `Other Content Types` in the navbar and observe the python packages listed. There should be "name", "version", and "filename" columns.
3. Check that these packages are searchable by filename.
4. Without the changes, there would only be "name" and "version" columns and you would not be able to distinguish between the files.
